data/company.yaml

```
@@ -0,0 +1,62 @@
+info:
+  founded_year: "2013"
+  employees: "25+"
+  headquarters: "Ukraine & USA"
+  business_registration: "LLC registered in Delaware, USA"
+  phone: "+1 754 216 9568"
+  email: "info@jetthoughts.com"
+  address: "Remote-first company with offices in Ukraine and USA"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)